### DISCUSSION TITLE
Использование YMapDefaultSatelliteLayer

### DISCUSSION THREAD

**Discussion by ShuraMeow** [yandex-maps-unofficial/vue-yandex-maps#131] [Q&A (2.x)] [answered]
Подскажите, пожалуйста, как воспользоваться DefaultSatelliteLayer. С обычным использованием Yandex JS API работают оба варианта: переопределение YMapDefaultSchemeLayer или наложение поверх дефолт слоя. Заранее благодарю.

**Comment by daniluk4000** [comment]
@ShuraMeow привет. А он хорошо работает? Я его тестировал и выглядел он как-то совсем печально, и сами Яндекс говорили, что спутника и гибрида в новых картах нет - вот я и не добавил такую функциональность.

Если оно работает, создайте пожалуйста вместо этой дискуссии Feature Request, и приложите:
- Скриншот, как должно выглядеть (один, если оба варианта ниже выглядят одинаково, и два, если по разному)
- Пример кода с "переопределением" SchemeLayer
- Пример кода с "наложением поверх"

Я возьму в реализацию. Спасибо!

**Reply by daniluk4000** [reply]
Пример кода имею в виду на чистом JS, вы явно его использовали (или с Vuefy, без разницы)

**Reply by daniluk4000** [reply, marked as answer]
@ShuraMeow пофиг короче, я сделал, выкатил в rc.8

**Reply by ShuraMeow** [reply]
Дай Бог здоровья, мил человек!
